nltk/collocations.py

```diff
@@ -122,6 +123,22 @@ def above_score(self, score_fn, min_score):
                 break
 
 
+
+def _build_new_documents(documents, window_size, pad_left=False, pad_right=False, pad_symbol=None):
```

Comment:
This is because the other function i.e. nltk.util.ngrams is using "None" as the padding symbol too, I just want to be consistent.